doc/build-unix.md

```diff
@@ -250,6 +250,24 @@ A list of additional configure flags can be displayed with:
     ./configure --help
 
 
+Setup and Build Example: Arch Linux
+-----------------------------------
+This example lists the steps necessary to setup and build a command line only, non-wallet distribution of the latest changes on Arch Linux:
+
+    pacman -S git base-devel boost libevent python
+    git clone https://github.com/bitcoin/bitcoin.git
+    cd bitcoin/
+    ./autogen.sh
+    ./configure --disable-wallet --without-gui --without-miniupnpc
```

Comment:
didn't need it for what i was doing. this was just from my notes. that's all. i also use `make -j 4 check` (it saves a lot of time to run multiple jobs at once) but didn't want to try to describe how to get the number of non-hyperthreaded cores in a generalized way.